constants/styles/StyledTextInput.ts

```diff
@@ -7,7 +7,8 @@ export const styles = StyleSheet.create({
         borderWidth: Layout.borderWidth,
         borderColor: Layout.borderColor,
         borderRadius: Layout.borderRadius,
-        paddingBottom: PixelRatio.getPixelSizeForLayoutSize(0.25)
+        paddingBottom: PixelRatio.getPixelSizeForLayoutSize(0.25),
+        marginVertical: 10
```

Comment:
Do we really need this? I mean it is a general component I would rather use `marginVertical` where you intended.

Reply:
I was looking for that too, but it did look good after this in all screens so I left it like this.

Reply:
But do you still know what the reason is for this change?

Reply:
To increase the spacing, it went to 0 with the keyboardawarescrollview

Reply:
Does KeyboardAwareScrollView ignore the `flex` values and pulls the StyledTextInputs together? I am afraid of this modification because it affects everything regardless of whether the input is in a scroll view or not. Please apply `marginVertical` only to those cases which are wrapped by a keyboard-awarescroll-view!

Reply:
Okay!

Reply:
It continued here: https://github.com/ivgraai/expiry-util/pull/107#issuecomment-674097142

What if we declare a property and we transfer it to the root view?